### ISSUE TITLE
Unable to build Bower mode project using the supported Node version

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#5892]
Trying to build a Bower/Vaadin 13 compatibility mode project the latest Node LTS (10.16.0) the build fails with
```
[Step 4/6] [INFO] Running 'gulp build_es6' in /opt/agent/work/a69251da2b2bb9e9/flow-tests/test-frontend-production-custom-context/target
[15:24:36][Step 4/6] [ERROR] internal/util/inspect.js:31
[15:24:36][Step 4/6] [ERROR] const types = internalBinding('types');
[15:24:36][Step 4/6] [ERROR]               ^
[15:24:36][Step 4/6] [ERROR] 
[15:24:36][Step 4/6] [ERROR] ReferenceError: internalBinding is not defined
[15:24:36][Step 4/6] [ERROR]     at internal/util/inspect.js:31:15
[15:24:36][Step 4/6] [ERROR]     at req_ (/opt/agent/work/a69251da2b2bb9e9/flow-tests/test-frontend-production-custom-context/target/node_modules/natives/index.js:137:5)
[15:24:36][Step 4/6] [ERROR]     at require (/opt/agent/work/a69251da2b2bb9e9/flow-tests/test-frontend-production-custom-context/target/node_modules/natives/index.js:110:12)
[15:24:36][Step 4/6] [ERROR]     at util.js:25:21```

Somehow related to Gulp versions and internal API used by Gulp: https://github.com/nodejs/node/issues/24985

**Comment by web-padawan** [comment]
Possible fix to make `gulp` 3.9.1 work with latest node versions: https://github.com/gulpjs/gulp/issues/2246#issuecomment-435646661

Personally, I would recommend upgrading from 3.9.1 to 4.0.0 or greater.

**Comment by Artur-** [comment]
Should be resolved by #5892 which just updates the lock file and use a newer version of `natives`
